benchmarks/aggregates.py

```diff
@@ -0,0 +1,85 @@
+import vaex
+
+from benchmarks.fixtures import generate_numerical
+
+
+class Aggregates:
+    pretty_name = "Performance of aggregates: stats, binby etc"
+    version = "1"
+
+    params = ([10**7, 10**8, 5*10**8, 10**9],)
```

Comment:
```suggestion
    params = ([10**7, 10**8, 5*10**8],)
```
I guess this doesn't run with 10**9?

Reply:
right, thanks for the catch! Will fix now.